src/main/java/org/opensearch/ad/transport/SearchAnomalyDetectorAction.java

```diff
@@ -15,11 +15,14 @@
 import org.opensearch.action.search.SearchResponse;
 import org.opensearch.ad.constant.CommonValue;
 
+import com.google.inject.Inject;
+
 public class SearchAnomalyDetectorAction extends ActionType<SearchResponse> {
     // External Action which used for public facing RestAPIs.
     public static final String NAME = CommonValue.EXTERNAL_ACTION_PREFIX + "detector/search";
     public static final SearchAnomalyDetectorAction INSTANCE = new SearchAnomalyDetectorAction();
 
+    @Inject
```

Comment:
Im curious about the purpose of adding an inject annotation to a no arg constructor

Reply:
Good call out. Removed it.